sdk/core/System.Net.ClientModel/tests/perf.internal/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Reflection;
+using Azure.Test.Perf;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using Perfolizer.Horology;
+
+if (!args.Contains("--bm"))
+    await PerfProgram.Main(Assembly.GetExecutingAssembly(), args);
```

Comment:
I'm assuming these are boilerplate across all perf projects that use our team's performance framework, but if not, we might come back and change this depending on the specific needs of this library.

Reply:
I'll have a look at what the boilerplate actually is to preserve that for future needs of the library, but I'll make sure we keep benchmarking capabilities easy.

Reply:
I went ahead and copied in the file from the Tables library, which I'm assuming is boilerplate code.  Feel free to modify it however you need for benchmarking the model stuff!